app/extensions/safe/ffi/ipc.js

```diff
@@ -270,7 +270,7 @@ const onSharedMDataDecision = ( e, data, isAllowed ) =>
         .catch( ( err ) =>
         {
             reqQ.req.error = err;
-            logger.error( err );
+            logger.error( 'Share MD decision error:: ', err );
```

Comment:
Cosmetic one, but perhaps we can start enhancing our error messages. I think here the error is not from a failed attempt to make any decision but when trying to encode a response, correct?